decidim-dev/lib/decidim/dev/test/rspec_support/capybara.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require "selenium-webdriver"
-require "system_test_html_screenshots"
+require "#{__dir__}/screenshot_helper"
```

Comment:
I think this require is unnecessary as the screenshot helper is already in the `rspec_support` folder which is required here:
https://github.com/decidim/decidim/blob/efeffe9c6211ca288c59ab75b8f9ba6d7a82f35f/decidim-dev/lib/decidim/dev/test/spec_helper.rb#L17